src/environments/environment.dev.ts

```diff
@@ -8,7 +8,7 @@ export const environment = {
 
   rpcUrl: 'https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/',
   chainId: ChainId.Volta,
-  cacheServerUrl: 'https://identitycache-dev.energyweb.org/v1',
+  cacheServerUrl: 'http://localhost:3000/v1',
```

Comment:
hmm, i think you need to revert this one line :)

Reply:
oh I don't know how I pushed this!